bpmn-model/src/main/java/io/camunda/zeebe/model/bpmn/util/VersionUtil.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright © 2017 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.camunda.zeebe.model.bpmn.util;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public final class VersionUtil {
```

Comment:
❓ Any specific reason why you copied this class instead of just laying a dependency from `bpmn-model` to `Zeebe-util`?

Reply:
We have another copy of `VersionUtil` inside `zeebe-client-java`. I thought that we don't want to depend on `zeebe-util` dependency just for one class.

Reply:
Wish I had an answer why this was done, but unfortunately I don't. Either way, I propose we use the one from Zeebe-util in this case if it's possible.